pages/docs/learn/versioning.mdx

```diff
@@ -0,0 +1,39 @@
+export const description = "Learn how ."
+
+# Versioning long running functions
+
+It's inevitable that long-running functions change over time. Inngest allows developers to implement multiple strategies for changing long-running code over time. In order to understand how to best manage changes over time, it's important to understand how the SDK implements determinism and [executes steps](/docs/learn/how-functions-are-executed).
```

Comment:
```suggestion
Long-running functions inevitably change over time. Inngest enables developers to implement multiple strategies for changing long-running code over time. To manage these changes effectively, it's crucial to understand how the SDK implements determinism and [executes steps](https://chatgpt.com/docs/learn/how-functions-are-executed).
```

- changes the first sentence to be clearer and more impactful
- solves redundant repetition in the last sentence

Reply:
```suggestion
ong-running functions inevitably change over time. Inngest enables developers to implement multiple strategies for changing long-running code over time. To manage these changes effectively, it's crucial to understand how the SDK implements determinism and [executes steps](/docs/learn/how-functions-are-executed).
```